pkg/crc/machine/kubeconfig.go

```diff
@@ -27,6 +27,20 @@ const (
 	developerContext = "crc-developer"
 )
 
+func updateClientCrtAndKeyToKubeconfig(clientKey, clientCrt []byte, srcKubeconfigPath, destKubeconfigPath string) error {
+	cfg, err := clientcmd.LoadFromFile(srcKubeconfigPath)
+	if err != nil {
+		return err
+	}
+	for username, auth := range cfg.AuthInfos {
+		if username == "admin" {
+			auth.ClientCertificateData = clientCrt
+			auth.ClientKeyData = clientKey
+		}
+	}
```

Comment:
Are you sure this works as expected? Isn't `auth` a copy of `cfg.AuthInfos["admin"]`? Changing `auth` would not modify `cfg.AuthInfos`?
https://stackoverflow.com/questions/42716852/how-to-update-map-values-in-go led me to try https://play.golang.org/p/iUGDFdnSAVr and ask these questions :)

Reply:
@cfergeau it is `map[string]*Auth` so https://play.golang.org/p/3Pomaiu7mfq is something which replicate it and it does change the value.

Reply:
Ah right, forgot to check the type of the values :) A pointer is good. Nit: not sure why we iterate over the map when we could be looking up `cfg.AuthInfos["admin"]` directly.